pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3406,6 +3418,7 @@ type StorageClassDeviceSet struct {
 // +kubebuilder:printcolumn:name="Pinning",type=string,JSONPath=`.status.info.pinning`,priority=1
 // +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=cephfsvg;cephsvg
```

Comment:
Should it technically have another `s` for consistency?
```suggestion
// +kubebuilder:resource:shortName=cephfssvg;cephsvg
```